src/components/ProjectSettingsTabs/GeneralTab.tsx

```diff
@@ -74,27 +85,21 @@ export const GeneralTab: React.FC<GeneralTabProps> = ({
   );
 };
 
-// TODO: Remove default values as part of EVG-15643
 const gqlToSchema = ({
-  enabled = false,
   owner,
   repo,
   branch,
   displayName,
-  batchTime = 0,
+  batchTime,
   remotePath,
   spawnHostScriptPath,
-  dispatchingDisabled = false,
-  deactivatePrevious = true,
-  repotrackerDisabled = false,
-  defaultLogger = "",
-  cedarTestResultsEnabled = false,
-  patchingDisabled = false,
+  defaultLogger,
   taskSync,
-  disabledStatsCache = false,
-  filesIgnoredFromCache = [],
-}) => ({
-  enabled: enabled ? "enabled" : "disabled",
+  ...data
```

Comment:
nit: can we either keep all of these variables spread and assign them to the schema like we do for owner and branch or access them from a data object?